tests/functional/tests/file_manager/suite_file_manager.cpp

```diff
@@ -0,0 +1,78 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstddef>
+
+#include "FATFileSystem.h"
+#include "FileManagerKit.h"
+#include "SDBlockDevice.h"
+#include "boost/ut.hpp"
+#include "filesystem"
+#include "tests/config.h"
+#include "tests/ft_utils.h"
+
+using namespace leka;
+using namespace boost::ut;
+
+SDBlockDevice sd_blockdevice(SD_SPI_MOSI, SD_SPI_MISO, SD_SPI_SCK);
+FATFileSystem fatfs("fs");
+
+void initializeSD()
+{
+	constexpr auto default_sd_blockdevice_frequency = uint64_t {25'000'000};
+
+	sd_blockdevice.init();
+	sd_blockdevice.frequency(default_sd_blockdevice_frequency);
+
+	fatfs.mount(&sd_blockdevice);
+}
+
+suite suite_file_manager_kit = [] {
+	auto file = FileManagerKit::File {};
+
+	const std::filesystem::path tmp_dir			  = "/fs/var/tmp";
```

Comment:
problem was that you were trying to create `/fs/tmp` as seen here

[`93146f1` (#1087)](https://github.com/leka/LekaOS/pull/1087/commits/93146f1225cb09c9e315ae26a633c84267b83c68#diff-58867dbb0e58ede1dd8017cc825a323ab8fbe7e19686f1a44420be6748475082R34)

It seems like it's not possible to create a new dir at the root of the filesystem and one can only use already created to directories to create more.

good news `/fs/var/tmp` is available!